css/main.css

```diff
@@ -325,3 +325,56 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-section {
+  font-family: "Inter", sans-serif;
+  background-color: white;
+  padding: 114px 7% 50px;
+}
```

Comment:
- [ ] According to [this Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), The `contact-us` section should be responsive. Please make sure this section has a mobile design as in the given Figma template.